inst/json/metadata_0.1.0.json

```diff
@@ -60,24 +60,24 @@
       { "id": "education", "label": "Education" },
       { "id": "education_closures", "label": "Closures" },
       { "id": "education_absences", "label": "Absences" },
-      { "id": "life_years", "label": "Life Years" },
+      { "id": "life_years", "label": "Life years" },
       { "id": "life_years_infants", "label": "Infants" },
       { "id": "life_years_adolescents", "label": "Adolescents" },
-      { "id": "life_years_working_age", "label": "Working-age Adults" },
-      { "id": "life_years_retirement_age", "label": "Retirement-age Adults" }
+      { "id": "life_years_working_age", "label": "Working-age adults" },
+      { "id": "life_years_retirement_age", "label": "Retirement-age adults" }
     ],
     "capacities": [
-      { "id": "hospital_capacity", "label": "Hospital Capacity", "description": "Total hospital beds" },
-      { "id": "icu_capacity", "label": "ICU Capacity", "description": "ICU hospital beds" }
+      { "id": "hospital_capacity", "label": "Hospital capacity", "description": "Total hospital beds" },
```

Comment:
@pratikunterwegs  Is the hospital capacity that we get as an output similarly about ‘spare’ capacity, as the input parameter is, or is it in fact a total capacity?